VERSIONING.md

```diff
@@ -0,0 +1,19 @@
+In order to support backwards and forwards compatibility, SBP messages
+are versioned and follow a small number of conventions, enabling
+devices to indicate understood message formats, capabilities and
+features. SBP uses a major, minor numbering scheme to indicate
+versions, where major and minor are represented by 8-bit values. The
+minor number is incremented by each message addition, deprecation. The
+major number is incremented when the format of a message is
```

Comment:
Change of a stable message. There's no provision here for changing the SBP framing structure. Should we include some language do disallow that?